### ISSUE TITLE
syncany-plugin-sftp: com.jcraft.jsch.JSchException: UnknownHostKey

### ISSUE THREAD

**Issue by jaltek** [syncany/syncany#524]
Hi,

I connected to my (already existing) syncany repo in my local network via the sftp-plugin. Anything went fine so far.
At the moment I'm on vacation and tried to connect to the repo from external by changing the ```hostname``` within the ```config.xml``` file with the following result:
```
$ sy ls-remote 
Checking remote changes ...Error: com.jcraft.jsch.JSchException: UnknownHostKey: foo.bar.domain. RSA key fingerprint is xx:xx:xx:xx:xx:xx:xx:xx:xx
Refer to help page using '--help'.
```
(Same with ```sy down``` )

I tried the following steps:

From my understanding syncany is unsing the ```known_host``` file in
``` 
~/.config/syncany/plugins/userdata/sftp
```
So I tried to
a.) add the entry from "my" known_host file (``` ~/.ssh/known_host ```)
b.) delete the known_host file

without success (same error).

In the source soce I saw that the client is set to
```
properties.put("StrictHostKeyChecking", "ask");
```
so i thought syncany should ask but it doesn't :)

**Comment by jaltek** [comment]
Ok - I think the problem is that syncany doens't ```ask``` again for an already connected repo.
On another machine that is also connect to the repo via SFTP I renamed the ```known_host``` file and got the same exception.

So I think syncany should ask every time - just think of moving the remote repo to another machine.

**Comment by schnittchen** [comment]
Is there a workaround?

**Comment by jaltek** [comment]
Did you tried to remove the plugin and add it to syncany again?

**Comment by schnittchen** [comment]
I don't think that would have worked. I initialized a new repo on the same server and removed it, then I had the necessary host entry.
